### PR TITLE
Docs: Add assigned parts to DG

### PR DESCRIPTION
**Pull request by Zer0Legion**
Features:
importExamScores
deleteShown
add
edit
delete

Manual testing
Launch & Shutdown
add
edit 
delete
deleteShown
list
importExamScores

planned enhancements
wrap text on result box so only 1 scrollbar needed
use both matric and email as primary key
make our sample data tags more relevant to head tutors
input validation for find command for prefixes other than lt and mt

### CODE REVIEW

**Comment by Zer0Legion** [comment]
Also fixed a bug in my code related to the " character in filepaths when I was doing manual testing.

File: src/main/java/seedu/address/logic/parser/ImportExamScoresCommandParser.java [hunk 1]
```diff
@@ -31,12 +32,17 @@ public ImportExamScoresCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, ImportExamScoresCommand.MESSAGE_USAGE));
         }
 
-        Path path = ParserUtil.parseFilePath(argMultimap.getValue(PREFIX_IMPORT).orElse(""));
-        if (!isCsvFile(path)) {
+        try {
```
Comment (Pughal77) [inline src/main/java/seedu/address/logic/parser/ImportExamScoresCommandParser.java:35]: Perhaps you can mention which issue this bug fixes?

**Review by Pughal77 — COMMENTED** [review]
Nice changes to the DG!

File: src/main/java/seedu/address/commons/util/CsvUtil.java [hunk 2]
```diff
@@ -36,6 +36,8 @@
 
 /**
  * Utility class for processing CSV files.
+ * Acknowledgement: The methods in this class were made possible with the help of
+ * the [OpenCSV](http://opencsv.sourceforge.net/) library.
```
Comment (Pughal77) [inline src/main/java/seedu/address/commons/util/CsvUtil.java:40]: thanks for adding this!

**Pull request merged**